ios/RNShopify.m

```diff
@@ -0,0 +1,133 @@
+
+#import "RNShopify.h"
+#import "Buy.h"
+
+@implementation RNShopify {
+  RCTPromiseResolveBlock _resolve;
+  RCTPromiseRejectBlock _reject;
+}
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_get_main_queue();
+}
+RCT_EXPORT_MODULE()
+
+RCT_EXPORT_METHOD(initialize:(NSString *)domain key:(NSString *)key)
+{
+
+  self.client = [[BUYClient alloc] initWithShopDomain:domain
+                                                     apiKey:key
+                                                      appId:@"8"];
+}
+
+RCT_EXPORT_METHOD(getShop:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+{
+
+  [self.client getShop:^(BUYShop *shop, NSError *error) {
+    if (shop && !error) {
+      resolve(shop.domain);
+    }
+    else {
+      reject(@"no_shop", @"There was no shop", error);
+    }
+  }]; 
+}
+
+RCT_EXPORT_METHOD(getProducts:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+{
+
+   [self.client getProductsPage:1 completion:^(NSArray *products, NSUInteger page, BOOL reachedEnd, NSError *error) {
```

Comment:
Yes, good catch. This was just for testing purposes at the time when I sent the review. But, it's already fixed in the new version and I'll include this change in the updated pull request.